### ISSUE TITLE
manifest file should be created along with sketch

### ISSUE THREAD

**Issue by codeanticode** [processing/processing-android#303]
Right now, the manifest file is created only after opening the permission selector, or running the sketch on the device or in the emulator. Should be available as soon the sketch is created, so the user can set parameters without having to using the permission selector or running it first.

**Comment by benfry** [comment]
As of recently, the editor now has the concept of a template project which might be helpful for this. No template code should be included, but a set of files can be added to the sketch by default.

**Comment by codeanticode** [comment]
Oh cool, thanks for the pointer. I will look into it!

**Comment by kfrajer** [comment]
Hi @codeanticode I was testing a sketch and I noticed I am able to add permissions using the permission selector. However, if I use the permission selector and un-check any permission, the manifest is not updated. I have to remove the manifest. Then, I open the permission selector and I will need to re-check all the permissions I need. Should I create a bug report?

**Comment by codeanticode** [comment]
@kfrajer Yes, please go ahead an open a separate issue about the permissions. Thank you!
